js/unix_formatting.js

```diff
@@ -314,6 +322,11 @@
             return output.join(''); //.replace(/\[\[[^\]]+\]\]/g, '');
         };
     })();
+
+    function from_ansi_formatting(input, settings) {
+        return $.terminal.from_ansi(input, settings.unixFormattingEscapeBrackets);
+    }
```

Comment:
Maybe better to don't have extra functions:

```javascript
    $.terminal.from_ansi = function(input, options) {
        ...
        if (options.unixFormattingEscapeBrackets) {
        }
    };
```

I'm using convention that argument is options and if I have defaults I use:

```javascript
var settings = $.extend({/* defaults*/}, options);
```

It would be nice if unix-formatting follow the same pattern.